src/components/navigation-desktop/navigation-desktop.component.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Link } from "react-router-dom";
+// Material UI
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+// SVG's
+import { ReactComponent as ProjectsIcon } from "../../assets/icons/projects.svg";
+import { ReactComponent as BulletJournalIcon } from "../../assets/icons/journal.svg";
+import { ReactComponent as Logo } from "../../assets/icons/sloth.svg";
+import { ReactComponent as ProfileIcon } from "../../assets/icons/profile.svg";
+import { ReactComponent as GuestIcon } from "../../assets/icons/guest.svg";
+
+const useStyles = makeStyles({
```

Comment:
Better practise is to name it simply 'styles'. Also, I would not recommend using makeStyles method that way. You can either create a simple object or an arrow function but only if you want to use the theme as parameter. In that case, for now, it should be a simple object. Next to make it work with component below you will also need to transform it a bit, I will give a short snippet of how it should look like. 

`import { withStyles } from '@material-ui/core/styles';
....
const styles = {
...
}

export const NavigationDesktop = withStyles(styles)(({classes}) => {     /* in that curly brackets you can put all the props you expect to get*/
...
return (
...
);
}
);`

Reply:
Searched the net a little bit and various resources are consistent on claiming that makeStyles > withStyles in the majority of cases https://github.com/mui-org/material-ui/issues/15351 <- Since I am not planning to create themed navigation I think that makeStyles should stay.

Reply:
I see the point, good research in that matter. But still you could at least swap this old function deceleration to const with arrow function.

Reply:
It's there to remind me that there are better ways to handle resizeListener https://medium.com/@renatorib/tackling-responsive-elements-in-react-and-why-ive-created-react-sizes-f7c87e3f9e64 (And also because ESLint had some kind of problem with arrow function in this place)